src/main/java/eu/europeana/metis/sandbox/domain/Event.java

```diff
@@ -4,58 +4,50 @@
 
 import eu.europeana.metis.sandbox.common.Status;
 import eu.europeana.metis.sandbox.common.Step;
-import java.util.Optional;
+import java.util.Collections;
+import java.util.List;
 
 /**
  * Event that contains a record and its processing details
  */
 public class Event {
 
   private final Record body;
+  private final List<RecordError> recordErrors;
```

Comment:
Since same record can have several errors (like media processing) now we collect them in this list

Reply:
`Record` and `List<RecordError>` together are your new object `RecordInfo`. Should we maybe have a `RecordInfo` instance here instead? Or perhaps have this class be a subclass of `RecordInfo`?

Reply:
Yes, I had it like that at the beggining then trying to follow [demeter law](https://medium.com/better-programming/demeters-law-don-t-talk-to-strangers-87bb4af11694) I change it (but it was a mistake to split the object here since all I needed to do was to add 2 get methods for each property) 
I will address this in [MET-2655](https://europeana.atlassian.net/browse/MET-2655)